zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,409 @@
+    def _expunge_oldest_task(self):
+        key, ts = '',  linux.get_current_timestamp()
+        for k in self.tasks:
+            task = self.tasks[k]
+
+            if task.is_running():
+                continue
+
+            if task.lastOpTime < ts:
+                key, ts = k, task.lastOpTime
+
+        if key != '': del(self.tasks[key])
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)